docs/bundles/SyliusFixturesBundle/custom_fixture.rst

```diff
@@ -0,0 +1,128 @@
+Custom fixture
+==============
+
+Basic fixture
+-------------
+
+Let's create a fixture that loads all countries from ``Intl`` library. We'll extend on ``AbstractFixture`` in order
```

Comment:
Let's create a fixture that loads all countries from **the** ``Intl`` library. We'll extend **the** ``AbstractFixture``